### ISSUE TITLE
[Accessibility : Keyboard & Navigation]  unable to move to next tab in header menu using "left and right arrow keys"

### ISSUE THREAD

**Issue by Nagamani71** [kimchi-project/kimchi#918]
![accessibility27](https://cloud.githubusercontent.com/assets/17590096/14145559/f35946d2-f6b1-11e5-9047-067960351abd.PNG)
